integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -1288,4 +1288,37 @@ protected JSONObject executeQuery(String query) throws IOException {
     Response response = client().performRequest(request);
     return new JSONObject(getResponseBody(response));
   }
+
+  @Test
+  public void testTimestampBracket() throws IOException {
```

Comment:
add a test with microseconds too
```
ts '2020-09-16 17:30:00.123'
```

Reply:
added in https://github.com/Bit-Quill/opensearch-project-sql/pull/297/commits/966efbb082b78fca450cd436641d3a3fe7a81842